### ISSUE TITLE
Dailymotion plugin : nominative address support

### ISSUE THREAD

**Issue by karlo2105** [streamlink/streamlink#733]
### Checklist

- [ ] This is a bug report.
- [ ] This is a plugin request.
- [x] This is a feature request.
- [ ] I used the search function to find already opened/closed issues or pull requests.

### Description

Dailymotion is a video platform used for live streaming and VOD. 

### Expected / Actual behavior

Currently Dailymotion plugin doesn't support nominative address for live streaming as Youtube does. 

### Reproduction steps / Stream URLs to test

Example : 
----------

BFMTV working
```
streamlink "http://www.dailymotion.com/video/xgz4t1_live-bfmtv_news" best
[cli][info] Found matching plugin dailymotion for URL http://www.dailymotion.com
/video/xgz4t1_live-bfmtv_news
[cli][info] Available streams: 128k (worst), 184p, 288p, 480p (best)
[cli][info] Opening stream: 480p (hds)
[cli][info] Starting player: C:\Users\Ddr\Downloads\VLCPortable\App\vlc\vlc
[cli][info] Player closed
[cli][info] Stream ended
[cli][info] Closing currently open stream...
```

BFMTV not working
```
streamlink "http://www.dailymotion.com/BFMTV" best
error: No plugin can handle URL: http://www.dailymotion.com/BFMTV
```

### Environment details (operating system, python version, etc.)

W7 PRO / Streamlink portable

**Comment by picozone** [comment]
Hi,

http://www.dailymotion.com/BFMTV is the media home page.
And http://www.dailymotion.com/video/xgz4t1_live-bfmtv_news is the stream URL.

Best regards,

**Comment by karlo2105** [comment]
On Youtube, media home page is covered unlike on DM : 

```
streamlink "https://www.youtube.com/user/Euronews" best
[cli][info] Found matching plugin youtube for URL https://www.youtube.com/user/E
uronews
[cli][info] Available streams: 144p (worst), 240p, 360p, 480p (best)
[cli][info] Opening stream: 480p (hls)
[cli][info] Starting player: C:\Users\Ddr\Downloads\VLCPortable\App\vlc\vlc
[cli][info] Player closed
[cli][info] Stream ended
[cli][info] Closing currently open stream...
```

**Comment by beardypig** [comment]
Fixed in #744.
